Soccar [UI]/src/nl/soccar/ui/fx/controller/GameResultsFXMLController.java

```diff
@@ -0,0 +1,130 @@
+    @FXML
+    private Button btnExitResults;
+    @FXML
+    private Button btnLogOut;
+    @FXML
+    private Label lblUsername;
+    @FXML
+    private Label lblCar;
+    @FXML
+    private TableView<ResultTableItem> tblBlueGoalsList;
+    @FXML
+    private TableView<ResultTableItem> tblRedGoalsList;
+
+    private static final Logger LOGGER = Logger.getLogger(GameResultsFXMLController.class.getSimpleName());
+
+    private Session currentSession;
+
+    private Player currentPlayer;
+
+    @Override
+    public void initialize(URL url, ResourceBundle rb) {
+        ClientController controller = ClientController.getInstance();
+        currentPlayer = controller.getCurrentPlayer();
+        currentSession = currentPlayer.getCurrentSession(); // Will never be null.
+
+        
+        btnLogOut.setOnAction(e -> Main.getInstance().logOut());
+        btnExitResults.setOnAction(e -> closeResults());
+        
+        tbclBlueUsername.setCellValueFactory(new PropertyValueFactory<>("username"));
+        tbclRedUsername.setCellValueFactory(new PropertyValueFactory<>("username"));
+        tbclBlueGoalsScored.setCellValueFactory(new PropertyValueFactory<>("goalsScored"));
+        tbclRedGoalsScored.setCellValueFactory(new PropertyValueFactory<>("goalsScored"));
+
+        lblUsername.setText(currentPlayer.getUsername());
+        lblCar.setText(currentPlayer.getCarType().toString());
+
+        setGameInfo();
+
+        Optional<SessionData> session = controller.getAllSessions().stream().filter(s -> s.getRoomName().equals(lblRoomName.getText())).findFirst();
+        if (!session.isPresent()) {
+            LOGGER.log(Level.WARNING, "An exception occured while getting the SessionData from the Game Server");
+            return;
+        }
+    }
+
+    /**
+     * Method that display the current settings of the room on the session view.
+     */
+    public void setGameInfo() {
+        Room room = currentSession.getRoom();
+        Game game = currentSession.getGame();
+
+        for(Player p : room.getTeamBlue().getPlayers()) {
+            int goals = 0;
+            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+            tblBlueGoalsList.getItems().add(r);
+        }
+      
+        for(Player p : room.getTeamRed().getPlayers()) {
+            int goals = 0;
+            ResultTableItem r = new ResultTableItem(p.getUsername(), goals);
+            tblRedGoalsList.getItems().add(r);
+        }
+        //TODO: count goals per username.
+        
+        lblRoomName.setText("Game Results - " + room.getName());
+        //TODO: Set endscore
+        
+    }
+
+    /**
+     * Method that navigates to the game view and set the application window to
+     * full screen mode.
+     */
+    public void startGame() {
+        Main main = Main.getInstance();
+        main.setScene(FXMLConstants.LOCATION_GAME_VIEW);
+        main.setFullScreen(true);
+    }
+    
+    public void closeResults() {

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Document this public method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://194.135.91.67:9000/coding_rules#rule_key=squid%3AUndocumentedApi)